doc/man7/EVP_KDF-KB.pod

```diff
@@ -136,9 +136,7 @@ NIST SP800-108, IETF RFC 6803, IETF RFC 8009.
 =head1 SEE ALSO
 
 L<EVP_KDF(3)>,
-L<EVP_KDF_CTX_new_id(3)>,
```

Comment:
What about this location here? Does it need to be removed, too?

https://github.com/openssl/openssl/blob/b62dcede015b0718e944df1c1d91b1e46eeb7deb/doc/man7/EVP_KDF-KRB5KDF.pod#L98

Reply:
Fixed in #10256, commit aa87a7347957874692f1b5ae41bfc864d89bfc69